api/libs/planning-units-grid/src/planning-unit-set.event.ts

```diff
@@ -9,5 +9,8 @@ import { IEvent } from '@nestjs/cqrs';
  *
  */
 export class PlanningUnitSet implements IEvent {
-  constructor(public readonly projectId: string) {}
+  constructor(
+    public readonly projectId: string,
+    public readonly planningUnitArea: number,
```

Comment:
Just to keep event purity, we should refrain from attaching more data than needed - it would require the publisher to know what to emit. I would say the BLM-handler should get the `planningUnitArea` value on its own.
(think of custom-grid in which we may don't even have this value)